macos/FluentUITestApp/TestButtonViewController.swift

```diff
@@ -0,0 +1,48 @@
+//
+// Copyright Microsoft Corporation
+//
+
+import AppKit
+import FluentUI
+
+
+class TestButtonViewController: NSViewController {
+	override func loadView() {
+		
+		let enabledButtons: [NSButton] = [
+			NSButton(title: "NSButton", target: nil, action: nil),
+			Button(title: "Primary Filled", style: .primaryFilled),
+			Button(title: "Primary Outline", style: .primaryOutline),
+			Button(title: "Borderless", style: .borderless),
+		]
+
+		let disabledButtons: [NSButton] = [
```

Comment:
There's avoidable duplication here... but maps hurt my head

Reply:
particularly because we follow up with another loop we probably should use a map

Reply:
what's the reason why adding NSButton?

Reply:
@harrieshin The NSButton on the top row was just for visual / size comparison..
I'll see if I can redo this with map